controllers/automation/packagedeployment_controller.go

```diff
@@ -207,6 +210,9 @@ func (r *PackageDeploymentReconciler) applyPackageMutations(ctx context.Context,
 					Namespace: n.GetNamespace(),
 				},
 			}
+			if id.Namespace == "" {
+				id.NameMeta.Namespace = "default"
```

Comment:
Not sure why this is needed?

The lookup in the cluster for the object should be GVK of the in-package injection resource, and Name, Namespace of the cluster.

Reply:
You're right, I removed it. Thanks!